device-types/Aruba/ArubaMC-3200.yaml

```
@@ -0,0 +1,29 @@
+manufacturer: Aruba
+model: Aruba3200
+slug: aruba3200
+is_full_depth: no
+u_height: 1
+interfaces:
+  - name: (BASE-T)gigabitethernet 0/0/0
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/0
+    type: 1000base-x-sfp
+  - name: (BASE-T)gigabitethernet 0/0/1
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/1
+    type: 1000base-x-sfp
+  - name: (BASE-T)gigabitethernet 0/0/2
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/2
+    type: 1000base-x-sfp
+  - name: (BASE-T)gigabitethernet 0/0/3
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/3
+    type: 1000base-x-sfp
```

```suggestion
  - name: gigabitethernet 1/0
    type: 1000base-t
  - name: gigabitethernet 1/1
    type: 1000base-t
  - name: gigabitethernet 1/2
    type: 1000base-t
  - name: gigabitethernet 1/3
    type: 1000base-t
```
Only list the default personality, not both for dual personality ports. Also my 3200s have the interfaces as 1/0, 1/1, 1/2, and 1/3.

---

it is complicated about "default personality" ;-) you are sure about interface ?

---

I'm just going off my 3200s for the interface names.
```
#show inventory 

Supervisor Card slot            : 0
System Serial#                  : redacted
SC      Assembly#               : 2010050C (Rev:10.00) 
SC      Serial#                 : redacted (Date:01/13/13) 
SC      Model#                  : Aruba3200-IL
<snip>
```
```
#show running-config | include 'interface gigabit' 
Building Configuration...
interface gigabitethernet 1/0
interface gigabitethernet 1/1
interface gigabitethernet 1/2
interface gigabitethernet 1/3
```